views/message-detail.ejs

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
This commit is a bugfix for the route, right? Should this file be committed here?

Reply:
Yes you're right, the files should not be here, I've removed them

Reply:
Thank you so much! Truly appreciate all your hard work, and hoping comments as a way of talking about our code has been comfortable for you. :)